### PR TITLE
Hide release indicators when there's no data (e.g. for kube)

### PR DESCRIPTION
**Pull request by stbenjam**
image

### CODE REVIEW

File: sippy-ng/src/releases/TopLevelIndicators.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    if (
+      props.indicators[indicator].current.runs !== 0 ||
+      props.indicators[indicator].previous.runs !== 0
+    ) {
+      noData = false
+    }
+  })
+  if (noData) {
+    return <></>
+  }
```
Comment (stbenjam) [inline sippy-ng/src/releases/TopLevelIndicators.js:49]: .

**Comment by deads2k** [comment]
wow, that's a lot of javascript.

/lgtm
/hold

**Comment by deads2k** [comment]
feel free to release the hold when appropriate.

**Comment by stbenjam** [comment]
> wow, that's a lot of javascript.
> 
> /lgtm
> /hold

Mostly just moving Top-level indicators to it’s own component

/hold cancel

**Pull request merged**